test/run_test.py

```diff
@@ -677,6 +704,60 @@ def run_doctests(test_module, test_directory, options):
     return result
 
 
+def print_log_file(test, file_path):
+    with open(file_path, "r") as f:
+        print_to_stderr("")
+        print_to_stderr(f'PRINT LOG FILE of {test} ({file_path})')
+        print_to_stderr(f'##[group]PRINT LOG FILE of {test} ({file_path})')
+        print_to_stderr(f.read())
+        print_to_stderr('##[endgroup]')
+        print_to_stderr(f"FINISHED PRINT LOG FILE of {test} ({file_path})")
+        print_to_stderr("")
+    os.remove(file_path)
```

Comment:
Hmm, this is minor, but I must say it looks weird in here, i.e. the print function actually removes the file after.  People won't expect that because print implies a read-only function

Reply:
I see what you try to do now, it's easier just remove the whole test-reports directory after testing though